test/e2e/typescript-build-output/typescript-build-output.test.ts

```diff
@@ -0,0 +1,23 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('typescript-build-output', () => {
+  const { next, isNextDeploy, isNextDev, skipped } = nextTestSetup({
+    files: __dirname,
+    skipStart: true,
+    skipDeployment: true,
+  })
+
+  if (skipped) {
+    return
+  }
+
+  if (isNextDeploy || isNextDev) {
+    it('should skip', () => {})
+    return
+  }
+
+  it('should show "Finished TypeScript" message in build output', async () => {
+    await next.start().catch(() => {})
```

Comment:
Is start needed here? Doesn't seem that it would be needed given the test is for the `next build`.